src/utils.js

```diff
@@ -18,8 +19,32 @@ function getPulumiStack() {
  * @param {string?} stack Pulumi stack
  * @return {{}} Environment key-value pairs
  */
-function getEnvironmentVariables(twilioClient) {
+function getEnvironmentVariables(twilioClient, args) {
+
   let envVars = process.env;
+
+  //remove recursion
+  if(args[0] !== "stack" && args[1] !== "ls") {
+
+    let environment = getPulumiStack();
+
+    if(environment){
+
+      const envFilePath = `${process.cwd()}/.${environment}.env`;
+
+      if (fs.existsSync(envFilePath)) {
+
+          const dotenv = require('dotenv').config({ path: envFilePath });
```

Comment:
I'm personally not a fan of having `require()` scattered around the file. It makes the code less manageable. Can you put this `require()` on the top of the file?

Reply:
Yep, me too. I was having some issues with dotenv and I forgot to add it to the top after my tests. Doing it now.